web/src/cypress/support/a11y.ts

```diff
@@ -0,0 +1,45 @@
+import axe from 'axe-core/axe.js'
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /** Test the accessibility of the current state of the page */
+      validateA11y: typeof validateA11y
+    }
+  }
+}
+
+// https://github.com/component-driven/cypress-axe/issues/118
+Cypress.Commands.add('injectAxe', () => {
+  cy.window({ log: false }).then((window) => {
+    const script = window.document.createElement('script')
+    script.innerHTML = axe.source
+    window.document.head.appendChild(script)
+  })
+})
+
+// no selector provided will result in the entire page being validated
+function validateA11y(selector = 'main[id="content"]'): void {
+  cy.window().then((win: Cypress.AUTWindow & { _axeInjected?: boolean }) => {
+    if (win._axeInjected) return
+    win._axeInjected = true
+    return cy.injectAxe()
+  })
```

Comment:
Should we combine these? put the `_axeInjected` check in `injectAxe` and then always call it from `validateA11y`, or just put the full contents in `validateA11y`

Reply:
yeah that's a good point- originally with us calling the lib version I had it separate but if we're going to grab the source ourselves we might as well combine them. 👍